src/targets/gpu/target.cpp

```diff
@@ -112,44 +112,36 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
     }
 
     // whiltelist supported Ops for the FP8 types
-    // different between fp8e4m3fnuz and OCP types because rocBLAS only has
-    // support for fp8e4m3fnuz
-    std::set<std::string> unsupported_fp8e4m3fnuz_ops = {};
-    if(not gpu::gfx_has_fp8fnuz_support())
+    // rocBLAS does not support any FP8 types
+    std::set<std::string> unsupported_fp8fnuz_ops = {};
+    if(string_value_of(MIGRAPHX_SET_GEMM_PROVIDER{}) == "rocblas")
     {
-        unsupported_fp8e4m3fnuz_ops.insert("dot");
-        unsupported_fp8e4m3fnuz_ops.insert("quant_dot");
+        std::cout << "rocBLAS provider selected. rocBLAS does not support FP8 types.\n";
```

Comment:
This will be printed even if fp8 is not used. If we print a warning, it should only print if fp8 is used. Although, I dont think we should warn here.

Reply:
Only print if user selected fp8.

Reply:
Depends on what we mean by used. There can be fp8 values in a pre-quantized model even if the user doesn't use our quantizer.

Reply:
Removed print based on discussion